File: SimpleCode/texteditor.py
import sys

def savetext(total_input):
    for line in total_input:
            open("testfile_python.txt", "a").write(line + '\n')

print("Welcome to Text Editor")
total_input = []
while 1:
    usrinput = input("")
    if usrinput == "exit":
        savetext(total_input)
        exit()
    elif usrinput == "save":
        savetext(total_input)
    else:
        total_input.append(usrinput)
        print(total_input)

File: SimpleCode/texteditor.c
#include <unistd.h>
#include <stdio.h>
#include <fcntl.h>

int main()
{
    char buffer[50];
    unsigned int i;
    int fd;

    printf("Welcome to Text Editor\n");
    i = 0;
    while(i < 50)
        buffer[i++] = 0;  
    read(0, buffer, 49);
    fd = open("testfile_c.txt", O_WRONLY|O_APPEND);
    i = 0;
    while(buffer[i])
        write(fd, &buffer[i++], 1);
    close(fd);
    return (0);
}
